components/html-parser.js

```diff
@@ -37,6 +39,26 @@ export const DEFAULT_INSTRUCTIONS = [
     processNode: (node, children) => {
       node.attribs.className = node.attribs.class;
       delete node.attribs.class;
+      delete node?.attribs?.style;
+
+      // Check if the link is a button by looking for the "btn" class or "btn-*" class
+      const isButton = node.attribs.className?.includes("btn");
+
+      if (isButton) {
+        const type = node.attribs.className?.match(/btn-(?:outline-)?(\w*)/)?.[1];
+        const map = {
+          primary: "red",
+          secondary: "blue",
```

Comment:
We may need to add `black` to this mapping. (seeing a black button changed to blue as part of a block with full html buttons inside of it)

/lang/commerce/